engine/orchestration/src/main/java/com/cloud/agent/manager/AgentManagerImpl.java

```diff
@@ -1862,4 +1871,49 @@ public void propagateChangeToAgents(Map<String, String> params) {
             sendCommandToAgents(hostsPerZone, params);
         }
     }
+
+    @Override
+    public void updateCapacityOfHosts() {
```

Comment:
@soreana can you simplify this method; i.e. extract some code to smaller methods.

Reply:
Done. :+1: